.env

```diff
@@ -30,7 +29,7 @@ CART_SERVICE_ADDR=cartservice:${CART_SERVICE_PORT}
 CHECKOUT_SERVICE_PORT=5050
 CHECKOUT_SERVICE_ADDR=checkoutservice:${CHECKOUT_SERVICE_PORT}
 
-CURRENCY_SERVICE_PORT=7000
```

Comment:
We have some colliding ports that were causing issues when running `docker compose up`

Reply:
@xoscar I understand that for development it's easier to use the localhost port with all services, but is it really required?
Changing the ports will require a lot of changes in most of the services and within the Loadgen.
I'd keep the ports as they are.

Reply:
I updated the ports as some of them were colliding but I have removed the bind to the localhost from the docker compose file

Reply:
Hey @xoscar, thanks for that!
@mic-max has exposed all ports to localhost in his PR to add tests to the project, so in the end we are back to your initial state. Sorry about that.

The main branch has the changes, and that may be one of the conflicts you will get later.